### ISSUE TITLE
select from values throwing

### ISSUE THREAD

**Issue by mmkal** [oguimbal/pgsql-ast-parser#1]
Hi - really awesome library. I'm using it to extract types from SQL queries, and I came across a query which throws an error:

```sql
select * from (values (1, 'one'), (2, 'two')) as vals (num, letter)
```

This is valid SQL and clients execute it fine, but it gets the following error when parsed by pgsql-ast-parser:

```
Error: Syntax error at line 1 col 16:

  select * from (values
                 ^
Unexpected word token: "values". Instead, I was expecting to see one of the following:

    - A "kw_select" token
```

The query is taken from postgres docs: https://www.postgresql.org/docs/9.4/queries-values.html

**Comment by mmkal** [comment]
Update: it might be being thrown off by the parentheses. This also fails with a similar error:

```sql
SELECT t1.id FROM (test_table t1 JOIN test_table t2 ON ((t1.id = t2.n)))
```

The above is how postgres pretty-formats a view created with an equivalent statement, so it'd be great if it were supported:

```sql
select t1.id from test_table t1 join test_table t2 on t1.id = t2.n
```

The error:

```
Error: Syntax error at line 1 col 21:

   SELECT t1.id FROM (test_table
                      ^
Unexpected word token: "test_table". Instead, I was expecting to see one of the following:

    - A "kw_select" token
```

**Comment by oguimbal** [comment]
Hi !
Thanks for the report 👍 
Both issues are fixed in pgsql-ast-parser@2.1.0
